ui/src/app/pages/workspace/workspace-share.tsx

```diff
@@ -12,7 +18,7 @@ import * as fp from 'lodash/fp';
 import * as React from 'react';
 
 import {
-  BillingAccountType,
+  BillingAccountType, Profile,
```

Comment:
nit: put new import on next line

Reply:
ill drop this change into my next follow up PR which is coming right up

Reply:
actually, my tests failed so I'll put them in here since I'm making changed anyways